torchvision/ops/misc.py

```diff
@@ -131,9 +132,12 @@ def __init__(
         activation_layer: Optional[Callable[..., torch.nn.Module]] = torch.nn.ReLU,
         dilation: int = 1,
         inplace: bool = True,
+        bias: Optional[bool] = None,
```

Comment:
bias is a bool, shouldn't the default be False?

Reply:
We want to keep BC so we can't have None be the default. Previously, bias was set to `norm_layer is not None`.